src/tests/ftest/container/full_pool_container_create.yaml

```
@@ -1,10 +1,11 @@
 # change host names to your reserved nodes, the
 # required quantity is indicated by the placeholders
 hosts:
-    test_machines1:
+    test_machines:
```

It would be good to make the move to the newer tag now
```suggestion
    test_servers:
```